### DISCUSSION TITLE
nautobot-environment-file/

### DISCUSSION THREAD

**Discussion by giscus** [jvanderaa/jvanderaa.github.io#198] [Announcements]
# nautobot-environment-file/

https://josh-v.com/nautobot-environment-file/

**Comment by Reidy10** [comment]
Thanks Josh, this helped me out a lot. I am still not too knowledgeable on Env Vars and expected that if i was exporting them in the nautobot py venv it would have been accessible to secret in reference in the gui, but adding the .env file reference to the nautobot.service helped resolve the problem for me!
